<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCamFeedWidget</class>
 <widget class="QWidget" name="VideoCamFeedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>111</width>
    <height>111</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	font: 75 12pt &quot;Comic Sans MS&quot;;
	border: 2px solid 	rgb(132, 93, 194)	 ;
	background-color: 	rgb(250, 179, 24)	;
	color: rgb(245, 246, 250);
	border-radius: 7px
}
QPushButton:hover{
	background-color: rgb(0, 201, 183)				;
}
QPushButton:pressed{
		background-color: rgb(0, 193, 174)		;}
#VideoCamFeedWidget{
	background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <widget class="MJPEGStreamer" name="videocam_feedW" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#videocam_feedW{
	background-color: rgb(78, 55, 115);
	border-radius:7px;
}</string>
   </property>
   <widget class="QPushButton" name="updateCamBtnW">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color: rgba(255,255,255,100);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fullscrnBtnW">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources/resources.qrc">
      <normaloff>:/images/fullscrnicon.png</normaloff>:/images/fullscrnicon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="hideShowSkeletonBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ะก</string>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MJPEGStreamer</class>
   <extends>QWidget</extends>
   <header>mjpegstreamer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
